tactics/auto.ml

```diff
@@ -394,8 +394,9 @@ and tac_of_hint dbg db_list local_db concl (flags, ({pat=p; code=t;poly=poly;db=
     | None -> mt ()
     | Some n -> str " (in " ++ str n ++ str ")"
     in
-    let sigma, env = Pfedit.get_current_context () in
-    pr_hint env sigma t ++ origin
+    (* XXX: Fix tclLOG *)
+    let env = Global.env () in
+    pr_hint env Evd.(from_env env) t ++ origin
```

Comment:
Note that using the global env vs the goal env might a priori have an impact on the qualification of names in printing!